src/app/pages/admin/twins/definitions/attributes/twins.attributes.component.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <em>{{value.created | date: 'long'}}</em>
+  <div *ngFor="let attr of value.attributes">
+    <b>{{attr.name}}</b>
+    <ul>
+      <li><b>Channel</b> {{ attr.channel }}</li>
+      <li><b>Subtopic</b> {{ attr.subtopic }}</li>
+      <li><b>Persisted</b> {{ attr.persist_state }}</li>
```

Comment:
Rm white spaces before `{{`

Reply:
Can't remove this space because there will be no separation between values and names of the values.

Reply:
In fact, will be fixed in the next commit.